content-src/components/Snippet/Snippet.scss

```diff
@@ -0,0 +1,77 @@
+.snippet {
+  $snippet-padding: 26px;
+  $snippet-bg-color: rgba($white, 0.95);
+  $snippet-divider-color: rgba($black, 0.1);
+  $snippet-width: 500px;
+  $snippet-base-font-size: 12px;
+  $snippet-title-size: 14px;
+  $snippet-image-size: 64px;
+  $snippet-image-margin: 15px;
+  $snippet-link-color: $link-blue;
+
+  display: flex;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background-color: $snippet-bg-color;
+  padding: $snippet-padding;
+  border-top: 1px solid $snippet-divider-color;
+  z-index: 1000;
+  align-items: center;
+
+  &.hidden {
```

Comment:
Looking at this, I see that I missed part of what's going on because I'm so used to a ".hidden" always meaning "visibility: hidden;" or "display: none;".   Maybe a different className here?